crates/trigger/src/cli.rs

```diff
@@ -64,12 +64,12 @@ where
         )]
     pub follow_components: Vec<String>,
 
-    /// Print all component output to stdout/stderr
+    /// Silence all component output to stdout/stderr
     #[clap(
-        long = "follow-all",
+        long = "silent",
```

Comment:
I'm a bit unsure about the name here - this promises _silence_, but users will still see Spin output such as "Available routes" links, and Spin logs per the Rust environment variable (errors by default).  I don't have any more constructive suggestions, though, sorry.

Reply:
`--silence-component-logs` ?

Reply:
with a short flag `-sh` like 🤫  ??

Reply:
`-sh` got a giggle out of me!

The long form might be _too_ long, though, and is a bit disconnected from the `follow` terminology used for following single components.  I am wondering about `follow-none` but I can't make myself love it.

Reply:
(for the avoidance of doubt, "got a giggle out of me" was intended to convey that I really liked it)

Reply:
How do you feel about `--silence-components`?

Reply:
`no-app-logs`?
`no-component-logs`?

I'd really love to use `sh`

Reply:
In isolation I'd probably go for `silence-components`.  But `no-{app|component}-logs` has the advantage that we could rename `follow` to `{app|component}-log` so you could have the related names we had with the `follow` family.

Let's wake @mikkelhegn up and make him have an opinion...

Reply:
Ok, hear me out: proposal I'm not feeling strongly about, so feel free to ignore.

The default (`spin up` with no flag outputs Spin logs + component logs to stdout / stderr.

`--log-files` / `--log-dir` / `--component-logs-<file-or-dir?>` sends the component logs to files / directory, and the Spin output (available routes, Spin errors) would still go to Spin's stdout / stderr.

Would something like this solve our issue?

Reply:
I think there is a world where someone may want to see the logs in the output AND save logs to a directory.

Reply:
If I've understood that right, the way to turn off component log output would be to specify `--log-dir`.  For me that is extremely non-obvious, sorry.  It also leaves us with the "follow only one" problem.

Reply:
`--quiet`/`-q` (no component logs) and `--quieter`/`-qq` (no spin logs either)